app/src/main/java/com/example/stretchy/database/dao/TrainingWithActivitiesDao.kt

```diff
@@ -17,6 +17,10 @@ interface TrainingWithActivitiesDao {
     @Query("SELECT * FROM training WHERE trainingId LIKE :id ")
     fun getTrainingsById(id: Long): TrainingWithActivitiesEntity
 
+    @Transaction
+    @Query("SELECT * FROM training_activities WHERE tId LIKE :id ")
+    fun getTrainingsActivitiesByTrainingId(id: Long): List<TrainingActivityEntity>
```

Comment:
change to `getTrainingsActivities`, change the `id` to `trainingId`, it will be more elegant